### DISCUSSION TITLE
which tool chains to cross compiling C project on the x64 liunx host for Termux?

### DISCUSSION THREAD

**Discussion by Jzhenli** [termux/termux-packages#15330] [Q&A] [answered]
Hi, I'm new to this project, and have a little question now.

I have a C project on the x64 linux PC, and now want to cross compiling it and build an executable code to run on the termux, but I don't know which tools chains should be used, would you share the successful experiences? 

Thanks

**Comment by sylirre** [comment, marked as answer]
We use Android NDK to cross compile software for Android OS.
